### PR TITLE
Added a test for `password_changed` with a custom validator

### PR DESCRIPTION
**Pull request by WouldYouKindly**

### CODE REVIEW

**Comment by WouldYouKindly** [comment]
@adamchainz

File: tests/auth_tests/test_validators.py [hunk 1]
```diff
@@ -57,6 +57,18 @@ def test_validate_password(self):
...
+    def test_password_changed_with_custom_validator(self):
+        class Validator:
+            def __init__(self):
+                self.call_count = 0
+
+            def password_changed(self, password, user):
+                self.call_count += 1
+
+        validator = Validator()
+        self.assertIsNone(password_changed('password', password_validators=(validator, )))
```
Comment (felixxm) [inline tests/auth_tests/test_validators.py]: Please remove unnecessary  space i.e. `(validator,)`.

Comment (WouldYouKindly) [reply tests/auth_tests/test_validators.py]: +

**Comment by timgraham** [comment]
The test could also check the values of the `password` and `user` arguments.

File: tests/auth_tests/test_validators.py [hunk 2]
```diff
@@ -57,6 +57,19 @@ def test_validate_password(self):
...
+    def test_password_changed_with_custom_validator(self):
+        class Validator:
+            def password_changed(self, password, user):
+                self.call_count = 1
+                self.password = password
+                self.user = user
+
+        validator = Validator()
+        self.assertIsNone(password_changed('password', password_validators=(validator,)))
+        self.assertEqual(validator.call_count, 1)
```
Comment (adamchainz) [inline tests/auth_tests/test_validators.py]: `call_count` isn't really necessary any more, as accessing `user` or `password` would be an `AttributeError` if the function hadn't been called, but this is a minor issue

Comment (WouldYouKindly) [reply tests/auth_tests/test_validators.py]: Right. I removed it.

**Comment by adamchainz** [comment]
✅

**Pull request merged**